visualizer/layout/connections.js

```diff
@@ -19,7 +19,7 @@ class Connection {
 
   // Avoid duplication of values so stats can be swtiched/recalculated with settings
   // If recalculating these proves to be a performance problem, consider caching values
-  getSourceRadius () { return this.scale.getCircleRadius(this.sourceLayoutNode.getWithinTime()) }
+  getSourceRadius () { return this.scale.getCircleRadius(this.originLayoutNode.getWithinTime()) }
```

Comment:
could `getSourceRadius` → `getOriginRadius` also be a good renaming candidate?

Reply:
@goto-bus-stop - Absolutely!